### ISSUE TITLE
[TS][Example] user-input e2e fails when run locally

### ISSUE THREAD

**Issue by chalin** [angular/angular.io#1762]
This is a known problem, but I haven't found an issue to track it, so here it is. When performing the tests manually, expected results are observed. So the problem might be with protractor's timing?

```
[14:52:11] AppDir: public/docs/_examples/user-input/ts
[14:52:16]   Suite: User Input Tests
[14:52:18]     passed - should support the click event
[14:52:18]     passed - should support the click event with an event payload
[14:52:18]     passed - should support the keyup event 
[14:52:19]     passed - should support user input from a local template let (loopback)
[14:52:20]     passed - should be able to combine click event with a local template var
[14:52:20]     failed - should be able to filter key events
[14:52:20]     Expected '' to equal 'abc'.
[14:52:21]     passed - should be able to filter blur events
[14:52:22]     passed - should be able to compose little tour of heroes
[14:52:22]   Suite failed: User Input Tests
```

**Comment by chalin** [comment]
@Foxandxss : FYI, still an issue as of 07/28.

**Comment by chalin** [comment]
Fixed by #2065.
